backend/app/serializers/api/v1/project_developer_serializer.rb

```diff
@@ -4,7 +4,10 @@ class ProjectDeveloperSerializer
       include JSONAPI::Serializer
 
       attributes :name, :slug, :about, :website, :instagram, :facebook, :linkedin, :twitter,
-        :mission, :project_developer_type, :categories, :impacts, :language
+        :mission, :project_developer_type, :categories, :impacts, :language, :picture_url,
+        :entity_legal_registration_number
```

Comment:
We need `review_status` as well. Public endpoints will return only `approved`

Reply:
added review_status to response ;)